Holovibes/sources/API.cc

```diff
@@ -1500,19 +1499,18 @@ void display_reticle(bool value)
     set_reticle_display_enabled(value);
 
     if (value)
-    {
-        UserInterfaceDescriptor::instance().mainDisplay->getOverlayManager().create_overlay<gui::Reticle>();
-        UserInterfaceDescriptor::instance().mainDisplay->getOverlayManager().create_default();
-    }
+        UserInterfaceDescriptor::instance().mainDisplay->getOverlayManager().enable<gui::Reticle>(false, 500);
```

Comment:
I feel like 500ms is a bit too short, for me 1s would be better
But if michael asked for it or something, its still just fine